airflow_connector.py

```diff
@@ -0,0 +1,55 @@
+"""Connects to the Cloud SQL database."""
+import os
+import MySQLdb
+
+
+class AirflowDB(object):
+  """"Provides the methods which allow interaction with the Airflow SQL database."""  # pylint: disable=line-too-long
+
+  def __init__(self, CLOUDSQL_CONNECTION_NAME, CLOUDSQL_USER, CLOUDSQL_PASSWORD, CLOUDSQL_HOST, CLOUDSQL_DB):
+    """Connects to the Cloud SQL database."""
+    # When deployed to App Engine, the `SERVER_SOFTWARE` environment variable
+    # will be set to 'Google App Engine/version'.
+    if os.getenv('SERVER_SOFTWARE', '').startswith('Google App Engine/'):
```

Comment:
don't look for env in the code, this gets ugly very fast. Pass in a boolean in your constructor instead.

Reply:
I'm a little confused. Are you saying that I should construct the DB connection in main.py and then pass it in to the AirflowDB object when I initialize it?